GOVERNANCE.md

```diff
@@ -0,0 +1,147 @@
+
+* Submit your code as a PR with the appropriate DCO sign-off.
+
+## Committers
+
+Project Committers have merge access on the Rez GitHub repository
+and are responsible for approving submissions by Contributors.
+
+### Committer Responsibilities
+
+Typical activities of a Committer include:
+
+* Helping users and novice contributors.
+
+* Ensuring a response to questions posted to the ASWF rez slack channel.
+
+* Contributing code and documentation changes that improve the project.
+
+* Reviewing and commenting on issues and pull requests.
+
+* Ensuring that changes and new code meet acceptable standards and are
+  in the long-term interest of the project.
+
+* Participation in working groups.
+
+* Merging pull requests.
+
+### How to Become a Committer
+
+Any existing Committer can nominate an individual making significant
+and valuable contributions to the Rez project to become a new
+Committer. New committers are approved by vote of the TSC.
+
+If you are interested in becoming a Committer, contact the TSC at:
+  rez-discussion@lists.aswf.io
+
+## Technical Steering Committee
+
+The Technical Steering Committee (TSC) oversees the overall technical
+direction of Rez, as defined in the project
+[charter](https://github.com/AcademySoftwareFoundation/foundation/blob/main/project_charters/rez-charter.pdf).
+This charter defines the TSC member terms and succession policies.
+
+The responsibilities of the TSC include:
+
+* Coordinating technical direction of the project.
+
+* Project governance and contribution policy.
+
+* GitHub repository administration.
+
+* Maintaining the list of additional Committers.
+
+* Appointing representatives to work with other open source or open
+  standards communities.
+
+* Discussions, seeking consensus, and where necessary, voting on
+  technical matters relating to the code base that affect multiple
+  projects.
+
+* Coordinating any marketing, events, or communications regarding the
+  project.
+
+The TSC elects a Chair person, who acts as the project manager,
+organizing meetings and providing oversight to project
+administration. The Chair is elected by the TSC. The Chair may
+also serve as the Rez representative on the Academy Software
+Foundation (ASWF) Technical Advisory Council (TAC), or one may
+be separately elected by the TSC. The Rez representative will
+represent the project at ASWF TAC meetings.
+
+### Current TSC Members
+
+Unless otherwise denoted with †, members of the TSC contribute to rez
+independent of their employer, but are listed regardless for transparency.
+
+* Allan Johns (chair, author) - NVIDIA
+* Brendan Abel - Walt Disney Imagineering
+* Jean-Christophe Morin - Signiant
+* Stephen Mackenzie - NVIDIA
+* Thorsten Kaufmann - Accenture Song Content
+
+### TSC Meetings
+
+All meetings of the TSC are open to participation by any member of the
+Rez community. Meeting times are listed in the [ASWF technical
+community calendar](https://lists.aswf.io/g/tac/calendar). The TSC
```

Comment:
I would point to https://lists.aswf.io/g/rez-discussion/calendar for the calendar since it's the rez calendar.

Reply:
done.